mhw_analysis/ncep/io.py

```diff
@@ -0,0 +1,51 @@
+""" Module to deal with I/O of CEMS-LENS files"""
+import os
+
+import numpy as np
+
+import iris
+
+
+def load_z500(dmy, path=None, end_dmy=None):
+    """
+    Load a Z500 cube from our NCEP nc file
+
+    Parameters
+    ----------
+    dmy : tuple  (day, month, year)
+        if end_date is not None, this is interpreted as the start date
+    path : str
+        Path to NCEP-DOE data
+    end_dmy : tuple, optional  (day, month, year)
+        if provided, scan for a range of dates
+        Note, this day is *not* inclusive
+
+    Returns
+    -------
+    z500_dmy : iris.Cube
```

Comment:
to my understanding, this will actually return a different object (CubeList I think) if the end_dmy is specified. This is important to note for debugging and implementation purposes.

Reply:
Still one cube, but 3 dimensions.  Have updated the doc